### DISCUSSION TITLE
Catastrophic forgetfullness

### DISCUSSION THREAD

**Discussion by balachander1964** [explosion/spaCy#10317] [Help: Coding & Implementations]
Hi Spacy Team, does the current version of Spacy handle the Catastrophic Forgetfullness automatically, or should we merge the predictions with new annotations when training? Awaiting your reply, Thank you. Bala

**Comment by polm** [comment]
No, spaCy doesn't do anything about catastrophic forgetting automatically. There are some topics about it in [the FAQ](https://github.com/explosion/spaCy/discussions/8226). 

If you need to retrain a model, you should have annotations for all the entity types. However we recommend training a component from scratch in most situations instead.
